src/kagglehub/handle.py

```diff
@@ -15,15 +15,15 @@
 NUM_UNVERSIONED_NOTEBOOK_PARTS = 2  # e.g.: <owner>/<notebook>
 
 
-@dataclass
+@dataclass(frozen=True)
```

Comment:
was `frozen=True` suggested automatically by the linter?

Reply:
No, this is required to make it a hashable type to use as key in the `VersionedDatasources` dictionary.